### PR TITLE
Update number.py

### PR DESCRIPTION
**Pull request by cinghialino**

### CODE REVIEW

File: custom_components/bambu_lab/number.py [hunk 1]
```diff
@@ -43,8 +43,8 @@ class BambuLabNumberEntityDescription(NumberEntityDescription, BambuLabTemperatu
         set_value_fn=lambda device, value: device.temperature.set_target_temp(TempEnum.NOZZLE, value)
     ),
     BambuLabNumberEntityDescription(
-        key="target_bed_temperature",
-        translation_key="target_bed_temperature",
+        key="bed_target_temperature",
+        translation_key="bed_target_temperature",
```
Comment (t3chguy) [inline custom_components/bambu_lab/number.py:47]: Looks like this will break translations

File: custom_components/bambu_lab/number.py [hunk 2]
```diff
@@ -43,8 +43,8 @@ class BambuLabNumberEntityDescription(NumberEntityDescription, BambuLabTemperatu
         set_value_fn=lambda device, value: device.temperature.set_target_temp(TempEnum.NOZZLE, value)
     ),
     BambuLabNumberEntityDescription(
-        key="target_bed_temperature",
-        translation_key="target_bed_temperature",
+        key="bed_target_temperature",
```
Comment (t3chguy) [inline custom_components/bambu_lab/number.py:46]: Looks like this makes it inconsistent with `target_nozzle_temperature`

Comment (cinghialino) [reply custom_components/bambu_lab/number.py:46]: sorry about that - I'm not a coder. the idea was to have more consistent naming 

`sensor.[printer]_bed_temperature` and `sensor.[printer]_bed_target_temperature` with `number.[printer]_bed_target_temperature`
or 
`sensor.[printer]_nozzle_temperature` and `sensor.[printer]_nozzle_target_temperature` with `number.[printer]_nozzle_target_temperature`

File: custom_components/bambu_lab/number.py (hunk 1 above)
Comment (AdrianGarside) [reply custom_components/bambu_lab/number.py:47]: The problem is that these changes (much as I want to make them) break people's existing dashboards.

**Comment by AdrianGarside** [comment]
Nice as this is, it breaks existing users' dashboards.

**Pull request closed without merging**